### DISCUSSION TITLE
sed command and \t for tabs

### DISCUSSION THREAD

**Discussion by tetsuo29** [holzschu/a-shell#698] [Q&A]
Hello!

I took a simple shell script that I have been using on my Mac for a few years now and modified it to run on a-shell.

It was a pretty easy conversion and almost everything is working.

However, the following sed command, which works just fine on macOS to convert CSV to Tab delimited text, is not placing tabs into the output but, it is instead placing lower case t’s:

`sed '/"/!s/,/\t/g;s/","/\t/g;s/"//g'`

I am no sed expert, and I admit that I did not write that command nor do I fully understand it. I found it long ago on some other discussion site and it has worked for years for me on macOS and Linux but now is failing on a-shell. Any insight as to why, or how it could be modified to work properly in a-shell?

**Comment by holzschu** [comment]
I would suggest trying with `\\t` instead of `\t`.

Meanwhile, I'll have a look at the command transmission between the keyboard and the command execution.

**Reply by tetsuo29** [reply]
I just got a chance to try that. Unfortunately it didn’t work. Still getting t’s instead of tabs in the output.
